roles/kubespray-defaults/defaults/main/download.yml

```
@@ -188,7 +188,7 @@ gvisor_containerd_shim_runsc_download_url: "{{ storage_googleapis_url }}/gvisor/
 nerdctl_download_url: "{{ github_url }}/containerd/nerdctl/releases/download/v{{ nerdctl_version }}/nerdctl-{{ nerdctl_version }}-{{ ansible_system | lower }}-{{ image_arch }}.tar.gz"
 containerd_download_url: "{{ github_url }}/containerd/containerd/releases/download/v{{ containerd_version }}/containerd-{{ containerd_version }}-linux-{{ image_arch }}.tar.gz"
 cri_dockerd_download_url: "{{ github_url }}/Mirantis/cri-dockerd/releases/download/v{{ cri_dockerd_version }}/cri-dockerd-{{ cri_dockerd_version }}.{{ image_arch }}.tgz"
-skopeo_download_url: "{{ github_url }}/lework/skopeo-binary/releases/download/{{ skopeo_version }}/skopeo-linux-{{ image_arch }}"
+skopeo_download_url: "{{ github_url }}/lework/skopeo-binary/releases/download/v{{ skopeo_version }}/skopeo-linux-{{ image_arch }}"
 yq_download_url: "{{ github_url }}/mikefarah/yq/releases/download/{{ yq_version }}/yq_linux_{{ image_arch }}"
```

Does this need to be changed?

```suggestion
yq_download_url: "{{ github_url }}/mikefarah/yq/releases/download/v{{ yq_version }}/yq_linux_{{ image_arch }}"
```

---

Good point, I'll add this one, not sure why I missed it.

Agree that the PR is large, any suggestion for splitting ? I'm reluctant to split the removing of the v-prefix, because that'd leave some components updatable with the old method and some with the new one...

---

Given your premise, I don't have a better idea of how to separate the PRs.
But I think for similar situations in the future, it should be possible to open an issue and have a todo list to track the status of the package updates.
Then about every three to five apps will send out a PR, which I think will be better.